fedimintd/src/ui/configgen.rs

```diff
@@ -90,13 +89,14 @@ fn trusted_dealer_gen(
         .map(|(&id, _)| {
             let id_u16: u16 = id.into();
             let peer = ServerPeer {
-                hbbft: ConnectionConfig {
-                    address: format!(
-                        "{}:{}",
-                        hostnames[id_u16 as usize].clone(),
-                        hbbft_base_port + id_u16
-                    ),
-                },
+                tls_cert: tls_keys[&id].0.clone(),
+                hbbft: format!(
+                    "{}:{}",
+                    hostnames[id_u16 as usize].clone(),
+                    hbbft_base_port + id_u16
+                )
+                .parse()
```

Comment:
This is a bit of an anti-pattern, the IP could be parsed and then `From<(IpAddress, u16)>` could be used to form the `SocketAddr`

Reply:
Turned these into Urls now.